Makefile

```diff
@@ -143,12 +143,10 @@ test-npm: $(NODE_EXE)
 	cd deps/npm ; npm_config_cache="$(shell pwd)/npm-cache" \
 	     npm_config_prefix="$(shell pwd)/npm-prefix" \
 	     npm_config_tmp="$(shell pwd)/npm-tmp" \
-	     ../../$(NODE_EXE) cli.js install --ignore-scripts
-	cd deps/npm ; npm_config_cache="$(shell pwd)/npm-cache" \
-	     npm_config_prefix="$(shell pwd)/npm-prefix" \
-	     npm_config_tmp="$(shell pwd)/npm-tmp" \
+	     ../../$(NODE_EXE) cli.js install --ignore-scripts && \
```

Comment:
Not cleaning up after running the tests will leave a __lot__ of stuff that isn't gitignored, which is why I added the new cleanup to begin with.